src/components/internal/MenuItem.tsx

```diff
@@ -167,9 +167,7 @@ function maybeWrapInLink(
       </span>
     </a>
   ) : (
-    <NavLink to={onClick} className="navLink">
-      {content}
-    </NavLink>
```

Comment:
**suggestion:** I think if we would've just changed this from `NavLink` to `Link` (both imported from `react-route-dom`), then this would work.

Reply:
Let me know if that does work. Happy to test it out with you!

Reply:
Actually, now I'm seeing that this is probably due to `onAction` happening here... not the link element itself. Going to try testing it as well.

Reply:
Hi @bdow 

I replaced NavLink with Link and works fine!, Please check the [Preview Branch](https://60466f7d43c37c0021788867-tvtxvdqrmf.chromatic.com/?path=/story/workspace-components-menu--basic-menu-items)